Source/WebKit/UIProcess/RemoteLayerTree/RemoteLayerTreeNode.mm

```diff
@@ -284,9 +284,13 @@
     m_effects->setEffects(WTFMove(clonedEffects));
     m_effects->setBaseValues(WTFMove(clonedBaseValues));
 
+#if PLATFORM(IOS_FAMILY)
+    m_effects->applyEffectsFromMainThread(layer(), host->animationCurrentTime());
+#else
     m_effects->initEffectsFromMainThread(layer(), host->animationCurrentTime());
+#endif
```

Comment:
I wonder if we should have a single method here on `RemoteAcceleratedEffectStack` and have `Mac` and `IOS` variants for that class which would do the _Right Thing_.